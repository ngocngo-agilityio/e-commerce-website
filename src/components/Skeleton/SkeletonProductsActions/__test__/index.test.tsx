// Libs
import { render } from '@testing-library/react';

// Components
import SkeletonProductActions from '..';

describe('SkeletonProductActions component', () => {
  test('should match snapshot', () => {
    const { container } = render(<SkeletonProductActions />);

    expect(container).toMatchSnapshot();
  });
});
