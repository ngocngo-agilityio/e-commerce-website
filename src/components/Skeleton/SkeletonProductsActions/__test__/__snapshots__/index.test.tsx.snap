// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonProductActions component should match snapshot 1`] = `
<div>
  <div
    class="css-120eo6q"
  >
    <div
      class="chakra-skeleton css-vdyamp"
    />
    <div
      class="chakra-skeleton css-vdyamp"
    />
    <div
      class="chakra-skeleton css-vdyamp"
    />
  </div>
</div>
`;
